server/routes/users.js

```diff
@@ -1,13 +1,15 @@
 const express = require('express');
 const router = express.Router();
+const { editUser } = require('../controllers/usersController.api')
 const { isAuth, userById } = require('../middlewares');
-const users = require('../controllers/usersController.api');
 
 
 router.get('/:id', (req, res) => {
   res.status(200).json(req.profile);
 });
 
+router.put('/users/:id', isAuth, editUser);
```

Comment:
Is the users portion is already living in index?

Reply:
Yeah, I fixed it! I may have missed things after I merged dev to this branch. Same with the /api/api that evan mentions later in this review